lang/scala/scala.py

```diff
@@ -8,32 +8,80 @@
 
 # Scala Common Types
 scala_common_types = {
-    "Byte": "Byte",
-    "Int": "Int",
-    "Double": "Double",
-    "Short": "Short",
```

Comment:
"short" seems to have been lost, is this intentional?

Reply:
D'oh, good catch! I amended this.